opensrp-opd/src/main/java/org/smartregister/opd/provider/OpdRegisterProvider.java

```diff
@@ -179,7 +179,8 @@ public void populatePatientColumn(CommonPersonObjectClient commonPersonObjectCli
         String childName = org.smartregister.util.Utils.getName(firstName, middleName + " " + lastName);
 
         String dobString = Utils.getDuration(opdRegisterProviderMetadata.getDob(patientColumnMaps));
-        //dobString = dobString.contains("y") ? dobString.substring(0, dobString.indexOf("y")) : dobString;
+        int year = dobString.contains("y") ? Integer.parseInt(dobString.substring(0, dobString.indexOf("y"))) : 0;
+        dobString = year >= 5 ? dobString.substring(0, dobString.indexOf("y")) : dobString;
```

Comment:
Make this translation proof by performing the calculation yourself and using the core method `Utils.getDuration` only if the years are for a 5 year old and below.

The `y` or `a` for french is dependent on the translation in the core library as you can see here https://github.com/OpenSRP/opensrp-client-core/blob/master/opensrp-app/res/values/strings.xml#L415

Reply:
@ekigamba The method `Utils.getDuration()` already returns translated year and month substring to make it translation proof I only had to remove the hard coded string *y* and use string resource *abbrv_years* - already available on opd any translation thereof can be extracted from the substring